ideascube/serveradmin/catalog.py

```diff
@@ -35,6 +35,32 @@ def rm(path):
         shutil.rmtree(path)
 
 
+def load_from_file(path):
+    """Load catalog data from a local file
```

Comment:
The fact it returns None is the important bit.

I removed the comment in the code where that function is used (https://github.com/ideascube/ideascube/pull/379/files/6b30174a5b4c57ac3405d1a1bd34b36cdffe284f#diff-e680b7b2bed4f6d8c15c2653df1eebdfL660), because I found it much better to have it documented once in the function.

Also, if we don't want these to be exposed as API, should I make them private?